lib/skinny_controllers/lookup/ensure_existence.rb

```diff
@@ -87,13 +78,7 @@ def policy_lookup(qualified_name)
     end
 
     def operation_lookup(qualified_name)
-      return if qualified_name.blank?
-      klass = qualified_name.safe_constantize
-
-      # Return if the constant exists, or if we can't travel
-      # up any higher.
-      return klass if klass
-      return unless qualified_name.scan(/::/).count > 1
```

Comment:
After some thought it seems that actually 7de5f89 is modifying the behavior in a wrong way. 
I reverted the commit and rebased.